app/_hub/kong-inc/http-log/index.md

```diff
@@ -66,33 +60,58 @@ params:
       required: true
       default:
       value_in_examples: http://mockbin.org/bin/:id
-      description: The HTTP endpoint (including the protocol to use) to which the data will be sent.
+      description: The HTTP endpoint (including the protocol to use) to which the data is sent.
     - name: method
       required: false
       default: "`POST`"
       value_in_examples: POST
       description: |
-        An optional method used to send data to the http server, other supported values are `PUT`, `PATCH`
+        An optional method used to send data to the HTTP server. Supported values are `POST` (default), `PUT`, and `PATCH`.
+    - name: content_type
+      required: true
+      default: "application/json"
+      value_in_examples: application/json
+      description: |
+        REVIEWERS: need a description.          
     - name: timeout
       required: false
       default: "`10000`"
       value_in_examples: 1000
-      description: An optional timeout in milliseconds when sending data to the upstream server
+      description: An optional timeout in milliseconds when sending data to the upstream server.
     - name: keepalive
       required: false
       default: "`60000`"
       value_in_examples: 1000
-      description: An optional value in milliseconds that defines for how long an idle connection will live before being closed
+      description: An optional value in milliseconds that defines how long an idle connection will live before being closed.
+    - name: flush_timeout
+      required:
+      default: "`2`"
+      value_in_examples: 2
+      description: |  
+        REVIEWERS: need a description.      
```

Comment:
Optional time in seconds.  If `queue_size > 1`, this is the max idle time before sending a log with less than queue_size records.

Reply:
Thank you for the description! Added in commit 2f431cb.